bids-validator/bids_validator/rules/session_level_rules.json

```diff
@@ -200,5 +200,30 @@
         "UNCInfant2V23"
       ]
     }
+  },
+
+  "microscopy_ses": {
+    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?\\1(_\\2)?(?:_sample-[a-zA-Z0-9]+)(?:_chunk-[a-zA-Z0-9]+)?(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+_)?(@@@_microscopy_ses_type_@@@)$",
```

Comment:
If I understand correctly, "sample" here is required. I think it would be more appropriate to have it optional at session level. For example, other modalities have all their entities except subject optional at that level so I think it's legit.